antlr/src/main/antlr4/org/apache/iotdb/db/qp/sql/SqlLexer.g4

```diff
@@ -827,7 +827,8 @@ MOD : '%';
 
 // Operators. Comparation
 
-OPERATOR_EQ : '=' | '==';
+OPERATOR_DEQ : '==';
```

Comment:
What's the real difference between `==` and `=` ? Are they alias mutually?

Reply:
if just use OPERATOR_EQ, parser can't tell the difference between "compare equal to" and "valuation".
for example:

select ts=1 and udf(ts,"id"="value") from root.test;

parser don't know whether "id"="value" means valuation or means a string equals to another string